File: Algorithm/Programmers/Level1/solution(21).py
# 정수 내림차순으로 배치하기

def solution(n):
    answer = 0
    nstr = str(n)
    nlist = sorted(nstr, reverse=True)
    nstr = ''
    for i in nlist:
        nstr += i
    answer = int(nstr)
    return answer

File: Algorithm/Programmers/Level1/solution(21).java
// 정수 내림차순으로 배치하기

import java.util.*;

class Solution {
    public long solution(long n) {
        long answer = 0;
        String[] arr = (String.valueOf(n)).split("");
        String result = "";
        Arrays.sort(arr);
        Collections.reverse(Arrays.asList(arr));
        for (String s : arr)
            result += String.valueOf(s);
        answer = Long.valueOf(result);
        return answer;
    }
}